reason_4.0/lib/core/minisite_templates/modules/events.php

```diff
@@ -3719,6 +3769,25 @@ function get_ical($events)
   		return $calendar -> get_icalendar_events();
 	}
 	/**
+	 * Get a json representation of a set of events
+	 * @param array $events entities
+	 * @return string JSON
+	 */
+	function get_json($events)
+	{
+		if(!is_array($events))
+		{
+			trigger_error('get_json needs an array of event entities');
+			return '';
+		}
+
```

Comment:
Is checking the truthiness of `get_value` fine, or should I use the `has_value` method (which checks against NULL) instead?